Gems/Atom/Feature/Common/Code/Source/CoreLights/QuadLightFeatureProcessor.cpp

```diff
@@ -227,5 +271,24 @@ namespace AZ
             return m_lightBufferHandler.GetElementCount();
         }
 
+        void QuadLightFeatureProcessor::UpdateBounds(LightHandle handle)
+        {
+            QuadLightData data = m_lightData.GetData<0>(handle.GetIndex());
+            LightCommon::LightBounds bounds = m_lightData.GetData<1>(handle.GetIndex());
```

Comment:
Should these be const& instead of copies?

Reply:
the first should be a const ref, the second should just be a ref - that was actually incorrect as written.